scripts/release.sh

```diff
@@ -55,6 +55,12 @@ safe_checkout_main() {
   fi
 }
 
+update_links() {
+  local version=$1
+  local versionTag="v${version}"
+  find ./content/docs/${version} -type f -exec sed -i "s|https://github.com/jaegertracing/jaeger/tree/main|https://github.com/jaegertracing/jaeger/tree/${versionTag}|g" {} \;
+}
```

Comment:
@yurishkuro is this what you had in mind? any thoughts on how I can test this?

Reply:
yes